### ISSUE TITLE
When updating settings, flash message is non-obvious

### ISSUE THREAD

**Issue by QuincyLarson** [freeCodeCamp/freeCodeCamp#16861]
Steps to reproduce:

1. Go to /settings
2. Update your email address
3. Nothing will seem to happen. It's only if you scroll to the top of the settings that you'll notice a confirmation flash message.

Screenshot:

![settings___freecodecamp](https://user-images.githubusercontent.com/985197/37191508-b7fd7070-2325-11e8-9afa-cd7238243db8.png)

**Comment by BerkeleyTrue** [comment]
This should be changed from a flash to a notification.

Note: flashes should be used for communications between a user and the backend (usually a fresh pageload is happening, but not always). We can also categorize notifications that need to be dismissed here. 

Toasts for notifications to the user from the app that do not need to be dismissed.

**Comment by YogeshSharma0201** [comment]
I looked into this issue, whenever user backend is updated a message is Toasted through makeToast() in update-user-epic.js by complete Observable, but for this to happen the payload passed to complete Observable should contain 'message' key so that message's value can be displayed. But when email is updated the payload passed contains 'info' instead of 'message' key  so the message object becomes undefined therefore no toast is shown.

I was able to fix it like this:
```
    const complete = actions$::ofType(types.updateUserBackend.complete)
    .flatMap(({ payload }) => {
      var message = payload.message || payload.info[0].split('\n').join(' ');
      return Observable.if(
          () => message.includes('project'),
          Observable.of(refetchChallengeMap(), makeToast({message: message})),
          Observable.of(makeToast({message: message}))
        );
      }
```
Screenshots:
![screenshot from 2018-03-13 19-44-56](https://user-images.githubusercontent.com/21025052/37363565-3d61a14a-271e-11e8-85d4-6917fa62d817.png)

![screenshot from 2018-03-14 00-04-57](https://user-images.githubusercontent.com/21025052/37363507-22dd0ecc-271e-11e8-8ff0-10a465e6895b.png)

Is this what was required? Have i done it correctly? If so i would like to open a PR.
